### ISSUE TITLE
WCAG: Make all PageBuilder blocks WCAG compliant

### ISSUE THREAD

**Issue by muratcakir** [smartstore/Smartstore#1335]

**Comment by mgesing** [comment]
The `HtmlBlock.TitleTag` property should be removed (marked as obsolete) because maintaining a consistent header order in accordance with WCAG is very difficult with it.
